LoopKit/PumpManager.swift

```diff
@@ -44,51 +47,77 @@ public protocol PumpManagerDelegate: PumpManagerStatusObserver {
     func pumpManagerRecommendsLoop(_ pumpManager: PumpManager)
 
     func startDateToFilterNewPumpEvents(for manager: PumpManager) -> Date
-
-    func startDateToFilterNewReservoirEvents(for manager: PumpManager) -> Date
 }
 
 
 public protocol PumpManager: DeviceManager {
     /// Rounds a basal rate in U/hr to a rate supported by this pump.
     ///
     /// - Parameters:
-    ///   - unitsPerHour: A desired rate of delivery in Units/hr
-    /// - Returns: a supported rate of delivery in Units/hr. The rate returned should not be larger than the passed in rate.
+    ///   - unitsPerHour: A desired rate of delivery in Units per Hour
+    /// - Returns: The rounded rate of delivery in Units per Hour. The rate returned should not be larger than the passed in rate.
     func roundToSupportedBasalRate(unitsPerHour: Double) -> Double
 
-    /// Rounds a bolus volume in U to a delivery amount supported by this pump.
+    /// Rounds a bolus volume in Units to a volume supported by this pump.
     ///
     /// - Parameters:
-    ///   - units: A desired rate of delivery in U
-    /// - Returns: a supported bolus volume in U. The rate returned should not be larger than the passed in rate.
+    ///   - units: A desired volume of delivery in Units
+    /// - Returns: The rounded bolus volume in Units. The volume returned should not be larger than the passed in rate.
     func roundToSupportedBolusVolume(units: Double) -> Double
 
-    // User selectable basal rates
+    /// All user-selectable basal rates, in Units per Hour
     var supportedBasalRates: [Double] { get }
 
-    // User selectable bolus volumes
+    /// All user-selectable bolus volumes, in Units
     var supportedBolusVolumes: [Double] { get }
```

Comment:
Worth documenting that `supportedBasalRates` and `supportedBolusVolumes` should be non-empty?